jobsets/default.nix

```
@@ -170,6 +171,13 @@ let
       prs = cardanoRosettaPrsJSON;
     };
 
+    cardano-rt-view = {
+      description = "RTView";
+      url = "https://github.com/input-output-hk/cardano-rt-view.git";
+      branch = "master";
+      prs = cardanoRTViewPrsJSON;
```

looks like the repo has bors.
```suggestion
      prs = cardanoRTViewPrsJSON;
      bors = true;
```

---

Yes, I need `bors` (as we have in `cardano-benchmarking` repo).